### DISCUSSION TITLE
Could not initialize class java.awt.Color

### DISCUSSION THREAD

**Discussion by codebylidia** [eclipse-birt/birt#1783] [Q&A]
Hello, could you please help me to find the root cause of the issue we are encountering.

We are using an old version of report plugins:
**org.eclipse.birt.runtime-3.7.2.v20120214-1408**
and we are getting following error message which fails to print the report.

Same implementation worked ok till now, till we encountered this issue.

```
java.lang.NoClassDefFoundError: Could not initialize class java.awt.Color
	at org.eclipse.birt.report.engine.nLayout.area.style.TextStyle.<init>(TextStyle.java:32) ~[org.eclipse.birt.runtime-3.7.2.v20120214-1408.jar:na]
	at org.eclipse.birt.report.engine.nLayout.area.impl.TextAreaLayout.buildTextStyle(TextAreaLayout.java:122) ~[org.eclipse.birt.runtime-3.7.2.v20120214-1408.jar:na]
...
```
[Error Log Sample.log](https://github.com/user-attachments/files/16118991/Error.Log.Sample.log)
Attached the full log error message.

What could be the reasons the  **java.awt.Color** is not loaded. Does it depends on system requirements, on some external plugin versions.
The used system is windows.
Do you have somewhere listed the requirements and dependencies for this birt plugin version ?

Thank you in advance
Kind Regards
Lidia

**Comment by merks** [comment]
java.awt.Color comes from the JRE/JDK you are using.  (Anything package starting with a `java` comes for the java runtime.)

**Reply by codebylidia** [reply]
Thanks @merks , yes I know that, does it mean that could not be related to Birt, and You can not help me here?

**Reply by merks** [reply]
It sounds like a problem with the Java runtime/version you are using.
